acceptance_tests/features/rh_ui_api_launch.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Launch a survey with a claims token from Respondent Home UI using a UAC
+
+  @reset_notify_stub
+  Scenario: Post a UAC to the launch endpoint and get redirected to a launch URL with a claims token
+    Given sample file "sample_1_limited_address_fields.csv" is loaded successfully
+
+    # Use an SMS fulfilment to obtain the plaintext UAC for the RH UI endpoint
```

Comment:
I'd remove this comment,  and the spacing.  The step "we retrieve the UAC and QID from the SMS fulfilment to use for launching in RH" and the scenario are well enough named